Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp

```diff
@@ -1096,6 +1098,23 @@ Vector<PropertyID> longhands_for_shorthand(PropertyID property_id)
                 return { };
         }
 }
+)~~~");
+
+    generator.append(R"~~~(
+// NOTE: This returns the fully-expanded longhands for a shorthand, excluding any intermediate longhands.
+Vector<PropertyID> expanded_longhands_for_shorthand(PropertyID property_id)
```

Comment:
This is a nice solution! Can you add it to the table in `Documentation/CSSGeneratedFiles.md`?